jest.config.js

```diff
@@ -5,6 +5,9 @@ module.exports = {
   // Automatically clear mock calls and instances between every test
   clearMocks: true,
 
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  // collectCoverageFrom: ['src/**/*.{js,jsx,mjs}'],
+
```

Comment:
What's with this commented out portion brov?

Reply:
when I rebased after you merged some earlier PR's my tests for components with images were failing. Dozie said jest couldn't parse the images. He helped me to configure it to what is there currently.

Reply:
What I'm saying is simple, remove this commented out portion if it doesn't need to be there.